pydoc/file_io/HDF5.md

```diff
@@ -0,0 +1,175 @@
+# HDF5
+
+HDF5 is an extremely flexible format. Because of this, it is important to adhere to these specifications in order for a file to be readable by Arkouda.
+
+More information on HDF5 is available [here](https://www.hdfgroup.org/solutions/hdf5/).
+
+## File Configuration
+
+Arkouda supports saving HDF5 files in 2 ways:
+
+- Single File
+  - All data is pulled local to the processing root node and saved into one file
+- Distributed Files (Default)
+  - Each file contains the portion of the data local to the locale where the file is being written are saved. This results in one file per locale.
```

Comment:
maybe reword `where the file is being written are saved`

Reply:
Done